webpack.config.js

```diff
@@ -28,9 +28,4 @@ config.module = {
   ],
 };
 
-// Overrides to disable CSS Modules for non-scss scripts, this means
-// CSS Modules will only be supported with .scss scripts
-config.cssRuleConfig.use = ["style-loader", "css-loader"];
-config.cssRuleConfig.test = /\.css$/;
```

Comment:
no need for these either (was needed for the old version of the ethiopian calendar)